core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -56,6 +58,21 @@ object ToResponse extends LowPriorityToResponseInstances {
     rep
   }
 
+  implicit def asyncStreamToResponse[A, CT <: String](implicit
```

Comment:
For now it looks like this:

```
curl http://localhost:8081/examples/3
{"x":0}{"x":1}{"x":2}
```

It works correctly for concatenated JSON case [https://en.wikipedia.org/wiki/JSON_Streaming](url)